res/controllers/Hercules-DJ-Console-RMX-scripts.js

```diff
@@ -450,6 +450,19 @@ HerculesRMX.Deck.prototype.cueSelectHandler = function(value) {
    }
 };
 
+HerculesRMX.Deck.prototype.sourceSelectHandler = function(value) {
```

Comment:
take a look at how Vestax 400 does pfl, for instance.  Notice that there is a binding for the control itself (which was learned through the midi wizard) and an output for turning on the light.